compiler/optimizer/Inliner.hpp

```diff
@@ -72,7 +72,6 @@ const float SECOND_BEST_MIN_CALL_FREQUENCY = .2275f; //65% of 35%
 
 class TR_BitVector;
 class TR_CallStack;
-class TR_ExtraAddressInfo;
```

Comment:
We seem to be removing TR_ExtraAddressInfo and using TR_ExtraValueInfo instead of TR_ExtraAbstractInfo. I assume these two changes are related in that TR_ExtraAddressInfo is the only other subclass of TR_ExtraAbstractInfo. Could you please elaborate on why this change is being made ?

Reply:
`TR_ExtraAddressInfo` has been removed here as it wasn't being used and its a small pain to forward declare, as its now a template. The other changes from `TR_ExtraAbstractInfo` to `TR_ExtraValueInfo` are due to the removal of `TR_ExtraAbstractInfo`. Rather than having to cast back and forth between the two classes, `TR_ValueInfo`'s methods now use `TR_ExtraValueInfo` directly.